content-learn/mastering-meshery-istio/introduction-to-service-meshes/linkerd/debugging.mdx

```diff
@@ -0,0 +1,62 @@
+---
+docType: "Chapter"
+chapterTitle: "Debugging"
```

Comment:
@Nikhil-Ladha Should I add "Optional" to `chapterTitle`?

Reply:
How about "Extra Credit"?

Reply:
Don't you think "Extra Credit" sounds more like a college word? :P

Reply:
`Optional` works for me. But, I will let @leecalcote have the final say on this.

Reply:
Let's go with `Optional` for now @Blakelist7 and get this PR in, we can update it later.

Reply:
ok cool